client/src/pages/RoomPage/index.tsx

```diff
@@ -309,7 +330,7 @@ function RoomPage({ name }: { name: string }): JSX.Element {
           onMuteClicked={toggleMute}
           onScreenShareClicked={handleClickScreenSharing}
           onLeaveClicked={handleLeaveClicked}
-          userInfoRef={selfUserInfoRef}
+          userInfo={userInfosRef.current[socket.id]}
```

Comment:
I tried it locally and seems to work with `selfInfo`

Reply:
I missed this nice catch. Yeah I think a ref is unnecessary here since it's being consumed by a react component and not a native class.